Update README.md (#1)
MindPort-GmbH/.github: profile/README.md

# MindPort GitHub Page
This is the home of VR Builder, an open-source toolkit for easily creating VR applications.
Here are some of the strengths of VR Builder.
- It empowers you to create fully functional VR apps—without writing code.
- Streamlines development by removing repetitive tasks to keep you focused.

> - Is the perfect solution for use cases such as VR training and product demos.

Reviewer: ```suggestion

VR Builder offers several key strengths:
- It empowers you to create fully functional VR apps—without writing code.
- It streamlines development by removing repetitive tasks to keep you focused.
- It is the perfect solution for use cases such as VR training and product demos.
```

  
## Repos
- [VR-Builder](https://github.com/MindPort-GmbH/VR-Builder): This is the official VR Builder open-source repository. VR Builder releases are available in the [Releases](https://github.com/MindPort-GmbH/VR-Builder/releases) section. The [VR Builder Pro](https://u3d.as/3pUD) asset on the Unity Asset Store is based on this, but offers additional functionality.

> - [VR Builder Documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation): This is the repo for the online documentation. The published documentation can be found [here](documentation.mindport.co).

Reviewer: ```suggestion
## Repos
- [VR-Builder](https://github.com/MindPort-GmbH/VR-Builder): This is the VR Builder open-source repository. You can find VR Builder releases in the [Releases](https://github.com/MindPort-GmbH/VR-Builder/releases) section. The [VR Builder Pro](https://u3d.as/3pUD) asset on the Unity Asset Store is based on this repository. It offers additional functionality.
- [VR Builder Documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation): This is the repository for the online documentation. The published documentation is available [here](documentation.mindport.co).
```

## Professional services

> If you want in scaling your VR application creation pipeline, please check out our [professional services](https://www.mindport.co/vr-builder/pricing) page.

Reviewer: ```suggestion
If you are looking to scale your VR application creation pipeline, please check out our [professional services](https://www.mindport.co/vr-builder/pricing) page.
```

## Contributions and Community

> We are committed to fostering an active and engaged community here on GitHub and on Discord. Here's some ways you could get involved.

Reviewer: ```suggestion
We are committed to fostering an active and engaged community here on GitHub and on Discord. Here are some ways you can get involved:
```

Engage on Discord: Share your pain points and success stories with us on [Discord](https://discord.gg/H7JXTTT9) and help each other.

> Issue reporting: Help us identify and fix issues by reporting them on [GitHub](https://github.com/MindPort-GmbH/VR-Builder/issues).

Reviewer: ```suggestion
Report issues: Help us improve by reporting bugs and issues on [GitHub](https://github.com/MindPort-GmbH/VR-Builder/issues).
```

> Improving [VR Builder](https://github.com/MindPort-GmbH/VR-Builder): Leave your mark on the project by enhancing code, documentation or visual elements.

Reviewer: ```suggestion
Contribute to [VR Builder](https://github.com/MindPort-GmbH/VR-Builder): Leave your mark on the project by enhancing code, documentation or visual elements.
```

> New Open-Source Documentation Repository: Participate in our open-source [documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation) to improve accessibility and usability.

Reviewer: ```suggestion
Contribute to documentation: Participate in our open-source [documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation) to improve accessibility and usability.
```

- [VR Builder Documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation): This is the repository for the online documentation. The published documentation is available [here](documentation.mindport.co).

## Professional services
If you are looking to scale your VR application creation pipeline, please check out our [professional services](https://www.mindport.co/vr-builder/pricing) page.

## Contributions and Community
We are committed to fostering an active and engaged community here on GitHub and on Discord. Here are some ways you can get involved:
Engage on Discord: Share your pain points and success stories with us on [Discord](https://discord.gg/H7JXTTT9) and help each other.
Report issues: Help us improve by reporting bugs and issues on [GitHub](https://github.com/MindPort-GmbH/VR-Builder/issues).
Contribute to [VR Builder](https://github.com/MindPort-GmbH/VR-Builder): Leave your mark on the project by enhancing code, documentation or visual elements.

> Contribute to documentation: Participate in our open-source [documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation) to improve accessibility and usability.

Reviewer: I think linking to the Repo will lead to nobody contributing. The way to improve the Dokumentation would be to click on "Improve this Doc" when on the documentation.
But I'm not sure how to communicate this in just one sentence.

An option would be to add a "##How to contribute" inside the readme.md of VR-Builder-Documentation. What d you think?

Reviewer: Made a change to the read me on https://github.com/MindPort-GmbH/VR-Builder-Documentation adding ## Contribute

> We are committed to fostering an active and engaged community here on GitHub and on Discord. Here are some ways you can get involved:
> Engage on Discord: Share your pain points and success stories with us on [Discord](https://discord.gg/H7JXTTT9) and help each other.
> Report issues: Help us improve by reporting bugs and issues on [GitHub](https://github.com/MindPort-GmbH/VR-Builder/issues).
> Contribute to [VR Builder](https://github.com/MindPort-GmbH/VR-Builder): Leave your mark on the project by enhancing code, documentation or visual elements.
> Contribute to documentation: Participate in our open-source [documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation) to improve accessibility and usability.

Reviewer: ```suggestion
We are committed to fostering an active and engaged community here on GitHub and on Discord. Here are some ways you can get involved:
- Engage on Discord: Share your pain points and success stories with us on [Discord](https://discord.gg/H7JXTTT9) and help each other.
- Report issues: Help us improve by reporting bugs and issues on [GitHub](https://github.com/MindPort-GmbH/VR-Builder/issues).
- Contribute to [VR Builder](https://github.com/MindPort-GmbH/VR-Builder): Leave your mark on the project by enhancing code, documentation or visual elements.
- Contribute to documentation: Participate in our open-source [documentation](https://github.com/MindPort-GmbH/VR-Builder-Documentation) to improve accessibility and usability.
```
